recipes/dydxprotocol-v4-proto-py/build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Python client
+if [[ "$(uname)" == "Darwin" ]]; then
+  sed -i '' "s/version=\"0.0.0\"/version=\"${PKG_VERSION}\"/g" v4-proto-py/setup.py
+else
+  sed -i "s/version=\"0.0.0\"/version=\"${PKG_VERSION}\"/g" v4-proto-py/setup.py
+fi
```

Comment:
Add `sed` as a build dependency to avoid this complication
```suggestion
sed -i "s/version=\"0.0.0\"/version=\"${PKG_VERSION}\"/g" v4-proto-py/setup.py
```

Reply:
Oh... I am such a rookie! lol